openedx/core/djangoapps/dark_lang/templates/dark_lang/preview_lang_include.html

```diff
@@ -3,10 +3,10 @@
 %>
 <%page expression_filter="h"/>
 
-<h1>
+<h3>
```

Comment:
From comment above: add class="mt-3" to add spacing without having to work with scss